src/app/json.-typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
```

Comment:
no idea what this is, does this allow you to load json files in a certain way or something?

Reply:
Yeah. It's how to be able to import json locally.  Otherwise this would be done in fulcrum